modules/host/persist_compat_1.0.0_test.go

```diff
@@ -0,0 +1,69 @@
+package host
+
+import (
+	"path/filepath"
+	"sync/atomic"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// TestHostPersistCompat100 tests breaking changes in the host persist struct
+// resulting from spelling errors. The test occurs by loading
+// hostpersist_compat_1.0.0.json, a v0.6.0 host persistence file that has been
+// pulled from the wild and adapted to have all non-zero values in its fields
+// for the purposes of testing.
+func TestHostPersistCompat100(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+	t.Parallel()
+	ht, err := newHostTester("TestHostPersistCompat100")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Close the host and then swap out the persist file for the one that is
+	// being used for testing.
+	ht.host.Close()
+	err = build.CopyFile("persist_compat_1.0.0.json", filepath.Join(ht.host.persistDir, settingsFile))
```

Comment:
the json file should be placed in a folder called `testdata`. The `go` tool will ignore such files. This is the convention used by the standard library, e.g. https://golang.org/src/image